lib/geoengineer/resources/aws_emr_cluster.rb

```diff
@@ -0,0 +1,20 @@
+########################################################################
+# AwsEmr is the +aws_emr_cluster+ terrform resource,
+#
+# {https://www.terraform.io/docs/providers/aws/r/emr_cluster.html}
+########################################################################
+class GeoEngineer::Resources::AwsEmrCluster < GeoEngineer::Resource
+  validate -> { validate_required_attributes([]) }
```

Comment:
should name be required?

Reply:
Is it below, where it validates it has the tag Name?